style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  background-image: url("image/Header bg@2x.jpg");
```

Comment:
Awesome using the background images as these images do not present any information but are for styling purposes. 

- [ ] Let's have more background images for the headline section as well resembling the Figma design.